from ..utils.exception import TestRestApiException


class AssertException(TestRestApiException):
    pass
